gobblin-runtime/src/main/java/org/apache/gobblin/runtime/api/MysqlMultiActiveLeaseArbiter.java

```diff
@@ -100,8 +100,9 @@ protected interface CheckedFunction<T, R> {
       + "PRIMARY KEY (flow_group,flow_name,flow_execution_id,flow_action))";
   private static final String CREATE_CONSTANTS_TABLE_STATEMENT = "CREATE TABLE IF NOT EXISTS %s "
       + "(epsilon INT, linger INT, PRIMARY KEY (epsilon, linger))";
-  private static final String GET_ROW_COUNT_STATEMENT = "SELECT COUNT(*) FROM %s";
-  private static final String INSERT_IN_CONSTANTS_TABLE_STATEMENT = "INSERT INTO %s (epsilon, linger) VALUES (?,?)";
+  // Only insert epsilon and linger values from config if this table does not contain a pre-existing values already.
+  private static final String INSERT_IN_CONSTANTS_TABLE_STATEMENT = "INSERT INTO %s (epsilon, linger) SELECT ?, ? "
+    + "WHERE NOT EXISTS (SELECT 1 FROM %s)";
```

Comment:
don't we need to overwrite in case the pre-existing values are out-of-date?  that's why I suggested `INSERT INTO... ON DUPLICATE KEY UPDATE`

Reply:
Fixed, had a misunderstanding about whether or not we want to update the values upon each redeploy if new config values are provided.